common/src/test/java/org/apache/sedona/common/raster/RasterOutputTest.java

```diff
@@ -44,6 +44,19 @@ public void testAsGeoTiff() throws IOException {
         assertEquals(rasterTest.getEnvelope().toString(), rasterOg.getEnvelope().toString());
     }
 
+    @Test
+    public void testToGeoTiff() throws IOException {
+        GridCoverage2D rasterOg = rasterFromGeoTiff(resourceFolder + "raster/test1.tiff");
+        String filePath = System.getProperty("user.dir") + "/../spark/common/src/test/resources/testToGeoTiffFunction/test1.tiff";
```

Comment:
The file path is wrong. It should be `System.getProperty("user.dir") + "/target/estToGeoTiffFunction/test1.tiff";`. Do not add content to the test resource folder.